release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -602,6 +602,11 @@ You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgrou
 
 You can enable swap memory use for {product-title} workloads on a per-node basis. For more information, see xref:../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-swap-memory_nodes-nodes-working[Enabling swap memory use on nodes].
 
+[id="ocp-4-10-node-maintenance-operator"]
+==== Place nodes into maintenance mode using the Node Maintenance Operator
+
+The Node Maintenance Operator (NMO) cordons off the node from the rest of the cluster, and drains all the pods from the node. By placing nodes under maintenance, you can investigate problems with a machine, or perform operations on the underlying machine, that might result in a node failure. This is a standalone version of NMO. If you have installed OpenShift Virtualization, then you must use the NMO which is bundled with it.
```

Comment:
```suggestion
The Node Maintenance Operator (NMO) cordons off nodes from the rest of the cluster and drains all the pods from the nodes. By placing nodes under maintenance, you can investigate problems with a machine, or perform operations on the underlying machine, that might result in a node failure. This is a standalone version of NMO. If you installed OpenShift Virtualization, then you must use the NMO that is bundled with it.
```

Reply:
Comments included

Reply:
Thanks for your review @kalexand-rh . All comments are included and the PR updated ready for merging

Reply:
@ogradyp Before requesting to merge this, please get it reviewed from @anna-savina and @razo7